src/leaflike/server.clj

```diff
@@ -1,15 +1,16 @@
 (ns leaflike.server
-  (:require [org.httpkit.server :as httpkit]))
+  (:require [org.httpkit.server :as httpkit]
+            [ring.middleware.json :refer [wrap-json-body]]
+            [leaflike.routes :as routes]))
 
-(defn app [req]
-  {:status  200
-   :headers {"Content-Type" "application/json"}
-   :body    "Welcome to Leaflike"})
+(defn app []
```

Comment:
The same defn vs def discussion is applicable here as well.

Reply:
was following convention. Your suggestion sounds about right too.